packages/compiler-core/src/transforms/hoistStatic.ts

```diff
@@ -116,7 +116,7 @@ export function isStaticNode(
         return false
       }
       const cached = resultCache.get(node)
-      if (cached !== undefined) {
+      if (cached !== void 0) {
```

Comment:
Are there any benchmarks for this being faster?

Reply:
@MichaelPan24 On old browsers is true, not anymore. Bear in mind the vue3 doesn't support old browsers.

> In modern browsers (JavaScript 1.8.5 / Firefox 4+), undefined is a non-configurable, non-writable property per the ECMAScript 5 specification. Even when this is not the case, avoid overriding it.

I don't think performance is the correct argument to use in here, `undefined` has been replaced for `void 0` in this repo, this is still a valid PR but the justification is misleading.

https://jsperf.com/void-0-vs-undefined-many-types

Reply:
@pikax , thanks for your correction so much,actually i also realized that the justification gonna be misleading